package/pyproject.toml

```diff
@@ -97,11 +97,13 @@ doc = [
 
 
 [project.urls]
-Documentation = 'https://docs.mdanalysis.org/'
-"User Guide" = 'https://userguide.mdanalysis.org/'
-"Issue Tracker" = 'https://github.com/mdanalysis/mdanalysis/issues'
-"User Group" = 'https://groups.google.com/g/mdnalysis-discussion/'
-Discord = 'https://discord.com/channels/807348386012987462/'
-Blog = 'https://www.mdanalysis.org/blog/'
+"Project URL" = "https://www.mdanalysis.org"
+"Download URL" = "https://github.com/MDAnalysis/mdanalysis/releases"
+Documentation = "https://docs.mdanalysis.org/"
+"User Guide" = "https://userguide.mdanalysis.org/"
+"Issue Tracker" = "https://github.com/mdanalysis/mdanalysis/issues"
+"Github Discussions" = "https://github.com/MDAnalysis/mdanalysis/discussions"
+Discord = "https://discord.com/channels/807348386012987462/"
+Blog = "https://www.mdanalysis.org/blog/"
 Twitter = 'https://twitter.com/mdanalysis'
```

Comment:
Add LinkedIn? And other platforms?

Also, Twitter is no more... Should this be changed to X?

Reply:
I removed twitter both here and the other PR, I'm hoping the rest should be stable enough.

Reply:
Yes, makes sense to remove socials!